lib/auth/sendVerificationRequest.ts

```diff
@@ -45,9 +45,86 @@ const html = (config: EmailConfig) => {
   const escapedEmail = `${email.replace(/\./g, '&#8203;.')}`;
 
   // HTML email template
+  // TODO: Replace logo URL with production URL
   return `
-        <p>Log into RCD APP platform as ${escapedEmail} by clicking the link below:</p>
-        <a href=${url}>Log into RCD</a>
+    <body
+      style="
+        background-color: #ffffff;
+        text-align: center;
+        font-family: Verdana, Arial', sans-serif;
+      "
+    >
+      <div
+        style="
+          height: 8px;
+          width: 100%;
+          background-color: #0B64CC;
+        "
+      />
+      <div
+        style="
+          padding: 32px 32px 40px;
+        "
+      >
+        <img
+          src="https://dev.rcdrichmondapp.ca/assets/logo.png"
+          style="
+            height: 65px;
+            width: 59px;
+            margin: 0 0 20px;
+          "
+          alt="Richmond Centre for Disability logo"
+        />
+        <div>
+          <h1
+            style="
+              color: #1A1A1A;
+              font-weight: bold;
+              font-size: 24px;
+              line-height: 150%;
+              margin: 0 0 20px;
+            "
+          >
+            Login
+          </h1>
+          <p
+            style="
+              font-size: 18px;
+              line-height: 150%;
+              margin: 0 0 32px;
+            "
+          >
+            Click the button below to authenticate as ${escapedEmail}
+            <br />
+            and sign into RCD APP Management System.
+          </p>
+          <a
+            href="${url}"
+            style="
+              background-color: #0B64CC;
+              color: #ffffff;
+              padding: 12px 24px;
+              border-radius: 6px;
+              font-size: 18px;
+              line-height: 150%;
+              text-decoration: none;
+            "
+          >
+            Log into RCD APP Management System
```

Comment:
i think this needs to be semi-bold from the design, could be wrong

Reply:
Good point. Thanks for the catch!